src/xdocs/config_naming.xml

```diff
@@ -903,6 +903,24 @@ class MyClass {
   final static int MYSELF = 100; // OK
   final static int myselfConstant = 1; // violation, name 'myselfConstant' must match pattern
                                        // '^log(ger)?$|^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'
+}
+        </source>
+        <p>
+         The following configuration is applicable by default.
+        </p>
+        <source>
+&lt;module name=&quot;ConstantName&quot;&gt;
+&lt;/module&gt;
```

Comment:
you do not need config example here. There is already config example at line 881, you only need to add code example to it.

Reply:
So I shall delete this and add the example below line 881?

Reply:
So what shall I delete or add over here? You want me to delete <source> and its content over here?

Reply:
remove config example and code example to existing default config example